### PR TITLE
[App Config] `appconfig kv import/export`: Add endpoint validation during import and export

### LINKED ISSUE #19139: az appconfig kv export usage error when using AD login auth
**Describe the bug**
`az appconfig kv export` to another appconfig instance always fails on a `usage error` when specifying the endpoint + auth-mode login params for both app config instances.

**Command Name**
`az appconfig kv export`

**Errors:**
```
usage error: --config-name NAME | --connection-string STR
```

**To Reproduce**
```
az appconfig kv export -d appconfig --endpoint https://myappconfiguration.azconfig.io --auth-mode login --dest-endpoint https://anotherappconfiguration.azconfig.io --dest-auth-mode login --key * --label * --preserve-labels
```

**Expected behavior**
This is a valid parameter set. App Config export should start 

**Environment summary**
azure-cli 2.27.0
Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Python (Windows) 3.8.9 (tags/v3.8.9:a743f81, Apr  6 2021, 13:22:56) [MSC v.1928 32 bit (Intel)]
Both in CMD and PsCore7 consoles.
Windows-10-10.0.19041-SP0
Installer: MSI

**Additional context**
This is documented as an example at 
https://docs.microsoft.com/en-us/cli/azure/appconfig/kv?view=azure-cli-latest#az_appconfig_kv_export-examples

Both Key Vaults have the `App Configuration Data Owner` Rbac role assigned for the authenticated azure-cli user.
The export does work when using access key authentication.

### PR DESCRIPTION
**Pull request by avanigupta**
**Description**

Fix #19139 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by avanigupta** [comment]
cc @shenmuxiaosen  for review.

File: src/azure-cli/azure/cli/command_modules/appconfig/_validators.py [hunk 1]
```diff
@@ -69,27 +69,31 @@ def validate_separator(namespace):
 def validate_import(namespace):
     source = namespace.source
     if source == 'file':
-        if namespace.path is None or namespace.format_ is None:
-            raise CLIError("usage error: --path PATH --format FORMAT")
+        if namespace.path is None:
+            raise RequiredArgumentMissingError("Please provide the '--path' argument.")
+        if namespace.format_ is None:
```
Comment (shenmuxiaosen) [inline src/azure-cli/azure/cli/command_modules/appconfig/_validators.py:74]: Can you run through our code base and refactor the general CLI error to explicit one? Like InvalidArgumentValueError, RequiredArgumentMissingError, MutuallyExclusiveArgumentError etc. Resolve this comment if you want to do that in a separete pr as this is mainly for bug fix.

**Comment by yonzhan** [comment]
App Config

File: src/azure-cli/azure/cli/command_modules/appconfig/_validators.py (hunk 1 above)
Comment (avanigupta) [reply src/azure-cli/azure/cli/command_modules/appconfig/_validators.py:74]: I only changed the error types that this PR was touching. I'll update all other CLIErrors in a separate PR.

**Review by shenmuxiaosen — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**